Source/Core/VideoBackends/D3D/VideoBackend.h

```diff
@@ -11,8 +11,8 @@ namespace DX11
 class VideoBackend : public VideoBackendBase
 {
 public:
-  bool Initialize(const WindowSystemInfo& wsi) override;
-  void Shutdown() override;
+  bool InitializeBackend(const WindowSystemInfo& wsi) override;
+  void ShutdownBackend() override;
```

Comment:
I think you forgot to rename `Shutdown` in `D3DMain.cpp` to match this.

Reply:
Yeah

I haven't fixed it yet because it doesn't work on Windows at all at the moment even with that fixed.  I should probably mark this as a draft until I get it working on Windows...